src/Components/v2/ArtworkFilter/index.tsx

```diff
@@ -0,0 +1,306 @@
+
+          <ArtworkGrid />
+        </Box>
+      </Media>
+
+      <Media greaterThan="xs">
+        <Flex>
+          <Box width="25%" mr={2}>
+            <ArtworkFilters mediums={filterMediums as any} />
+            <Separator mb={2} />
+          </Box>
+          <Box width="75%">
+            <Box id="jump--searchArtworkGrid" />
+
+            <ArtworkGrid />
+          </Box>
+        </Flex>
+      </Media>
+    </Box>
+  )
+}
+
+const ArtworkFilterFragmentContainer = createRefetchContainer(
+  ArtworkFilter,
+  {
+    viewer: graphql`
+      fragment ArtworkFilter_viewer on Viewer
+        @argumentDefinitions(
+          aggregations: {
+            type: "[ArtworkAggregation]"
+            defaultValue: [MEDIUM, TOTAL]
+          }
+
+          acquireable: { type: "Boolean" }
+          artist_id: { type: "String" }
+          at_auction: { type: "Boolean" }
+          attribution_class: { type: "[String]" }
+          color: { type: "String" }
+          for_sale: { type: "Boolean" }
+          height: { type: "String" }
+          inquireable_only: { type: "Boolean" }
+          keyword: { type: "String!", defaultValue: "" }
+          major_periods: { type: "[String]" }
+          medium: { type: "String" }
+          offerable: { type: "Boolean" }
+          page: { type: "Int" }
+          partner_id: { type: "ID" }
+          price_range: { type: "String" }
+          sort: { type: "String", defaultValue: "-partner_updated_at" }
+          width: { type: "String" }
+        ) {
+        filter_artworks(aggregations: $aggregations, size: 0) {
+          aggregations {
+            slice
+            counts {
+              name
+              id
+            }
+          }
+        }
+        filtered_artworks: filter_artworks(
+          aggregations: [TOTAL]
+          medium: $medium
+          major_periods: $major_periods
+          partner_id: $partner_id
+          for_sale: $for_sale
+          at_auction: $at_auction
+          acquireable: $acquireable
+          offerable: $offerable
+          inquireable_only: $inquireable_only
+          size: 0
+          sort: $sort
+          price_range: $price_range
+          height: $height
+          width: $width
+          artist_id: $artist_id
+          attribution_class: $attribution_class
+          color: $color
+          keyword: $keyword
+          page: $page
+        ) {
+          ...ArtworkFilterArtworkGrid2_filtered_artworks
+        }
+      }
+    `,
+  },
+  ArtworkQueryFilter
+)
+
+export const ArtworkFilterQueryRenderer = ({ term = "andy warhol" }) => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <FilterContextProvider keyword={term} {...initialFilterState}>
+      <QueryRenderer<ArtworkFilterQueryType>
+        environment={relayEnvironment}
+        // FIXME: Passing a variable to `query` shouldn't error out in linter
+        /* tslint:disable:relay-operation-generics */
+        query={ArtworkQueryFilter}
```

Comment:
@alloy - noticed that the linter doesn't like variables passed to `query`. Was there a specific reason for this?

Reply:
Unsure what you mean by “passing a variable to `query`”. Can you provide an example and the error?

Reply:
He's referring to the variable named `ArtworkQueryFilter`. It's a variable because the same query is used twice, in both the fragmentContainer and queryRenderer. Instead of copy-pasting the same query and naming it different things, he's reusing the query definition in two places (here and https://github.com/artsy/reaction/pull/2572/files/9555125ef04e9af1f23a45990ec41cd893b94341#diff-822e8fa213e43cff5a5a98e1ce82eaeaR223).

The error without that `lint:disable` line is `expected a graphql`…` tagged-template tslint(relay-operation-generics)`.

Reply:
Yup, the linter is looking for 

```
query={graphql` ... `}
```
or at least a "`".